### PR TITLE
Network connectivity tests - Add new IPv6 Computes examples

### PR DESCRIPTION
**Pull request by msampathkumar**
Add new IPv6 Computes samples for Network connectivity tests

Note: This is recently launched feature.

### CODE REVIEW

File: network/networkmanagement/ipv6_examples/main.tf [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  name    = "example-allow-incoming-ipv4"
+  network = google_compute_network.default.name
+
+  allow {
+    protocol = "all"
+  }
+  source_ranges = ["10.0.0.0/22"]
+}
+
+data "google_compute_image" "default" {
```
Comment (betsy-lichtenberg) [inline network/networkmanagement/ipv6_examples/main.tf:113]: Consider moving this earlier in the sample, before where it's referred to. Maybe at line 32?

Comment (msampathkumar) [reply network/networkmanagement/ipv6_examples/main.tf]: Good idea. Thank you.

**Comment by betsy-lichtenberg** [comment]
This passes when I run the test locally, so I'm not sure why it's failing after 2 minutes. Kicking off tests again.

File: network/networkmanagement/ipv6_examples/main.tf [hunk 2]
```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2023 Google LLC
```
Comment (betsy-lichtenberg) [inline network/networkmanagement/ipv6_examples/main.tf:2]: Please change to flat structure. Something like

network_management/ipv6_dual_stack/main.tf

**Pull request merged**